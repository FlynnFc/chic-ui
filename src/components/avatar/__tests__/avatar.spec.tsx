import Avatar from '..';
import { render } from '../../../test/utils';

describe('Avatar Component', () => {
  it('Renders Correctly', () => {
    const AvatarGroup = render(
      <>
        <div>
          <Avatar src="chic_copy.png" height={120} width={100} />
          <Avatar src="chic_copy.png" isActive />
        </div>
        <div>
          <Avatar name="KS" />
          <Avatar name="KS" isActive />
        </div>
      </>
    );
    expect(AvatarGroup).toMatchSnapshot();
  });
});
