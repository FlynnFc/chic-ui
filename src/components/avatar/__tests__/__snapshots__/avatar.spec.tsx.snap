// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Component Renders Correctly 1`] = `
Array [
  <div>
    <span
      className="sc-bdfBQB gvmZcQ"
      height={120}
      src="chic_copy.png"
      width={100}
    >
      <img
        className="sc-gsTEea bnGJKc"
        src="chic_copy.png"
      />
    </span>
    <span
      className="sc-bdfBQB fWIYqC"
      height={80}
      src="chic_copy.png"
      width={80}
    >
      <img
        className="sc-gsTEea bnGJKc"
        src="chic_copy.png"
      />
    </span>
  </div>,
  <div>
    <span
      className="sc-bdfBQB cKpsBK"
      height={80}
      name="KS"
      width={80}
    >
      <span
        className="sc-dlfnuX ibWqje"
      >
        KS
      </span>
    </span>
    <span
      className="sc-bdfBQB fWIYqC"
      height={80}
      name="KS"
      width={80}
    >
      <span
        className="sc-dlfnuX ibWqje"
      >
        KS
      </span>
    </span>
  </div>,
]
`;
